cnxepub/models.py

```diff
@@ -185,7 +196,7 @@ def flatten_to(model, flatten_filter):
     for m in flatten_model(model):
         if flatten_filter(m):
             yield m
-    raise StopIteration()
+    return
```

Comment:
I see you're changing `raise StopIteration()` to `return`, maybe we don't even need `return`...?